tools/vipps-partner-api-postman-collection.json

```diff
@@ -37,34 +38,12 @@
 							{
 								"description": "(Required) Vipps Subscription Key",
 								"key": "Ocp-Apim-Subscription-Key",
-								"value": "da7d5b0e18a84aeda961c0c31b75c2a9"
+								"value": "{{Ocp-Apim-Subscription-Key}}"
 							},
 							{
 								"description": "(Required) Idempotency key for the request, ensures idempotent actions. Not required, but recommended.",
 								"key": "Idempotency-Key",
-								"value": "81b83246-5c19-7b94-875b-ea6d1114f099"
-							},
-							{
-								"key": "Vipps-System-Name",
```

Comment:
I think these four headers are supposed to remain for each endpoint, because I was asked to add them to all the other postman examples:
Vipps-System-Name
Vipps-System-Version
Vipps-System-Plugin-Name
Vipps-System-Plugin-Version

Reply:
Yeah... Was thinking about it, but it was just imported from openapi spec, and we don't include it there :/